langkit/documentation.py

```diff
@@ -686,6 +686,45 @@ def instantiate_templates(doc_dict: Dict[str, str]) -> DocDatabase:
         buffer for a requested filename.
     """,
 
+    #
+    # Event handlers
+    #
+    'langkit.create_event_handler': """
+        Create a event handler. When done with it, the result must be passed
```

Comment:
an* event (I noticed other occurences in the patch)

Reply:
aaah right, I wrote the docs and did the renaming signal -> event afterwards, so not surprising...